app/views/bookmarks/list.html.erb

```diff
@@ -82,7 +82,9 @@
 <br />
 
 <%= link_to 'New bookmark', :action => 'new', :id => @topic.id %> |
-<%= link_to 'Back', :controller=> 'sign_up_sheet', :action=>'list', :assignment_id => @topic.assignment_id %>
+<%= link_to 'Back', '#', :onclick => 'pageBack()' %>
```

Comment:
Just curious, what does the pound sign do here?

Reply:
It's used when you want to link back to the page itself. It fails to work otherwise for some reason